app/services/smtp.py

```
@@ -0,0 +1,89 @@
+                password=settings.smtp_password,
+                sender=envelope_from,  # This sets the MAIL FROM (envelope sender)
+                recipients=all_recipients,
+                start_tls=True,  # Force STARTTLS
+                validate_certs=True,
+                timeout=30
+            )
+            
+            logger.info(f"Email sent successfully to {all_recipients}")
+            
+        except aiosmtplib.SMTPException as e:
+            logger.error(f"SMTP error sending email: {str(e)}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)